packages/types/src/DidResolver.ts

```diff
@@ -55,7 +55,7 @@ export interface IDidResolver {
    * @returns A promise of a [[DidResolvedDetails]] object if the didUri contains no fragment, [[ResolvedDidKey]] or [[ResolvedDidServiceEndpoint]] otherwise. Null if a resource cannot be resolved.
    */
   resolve: (
-    didUri: DidUri | DidResourceUri | DidPublicServiceEndpoint['uri']
+    didUri: DidUri | DidResourceUri | DidPublicServiceEndpoint['id']
```

Comment:
for did and did resources we enter a uri, but for service endpoints just the id? Seems inconsistent

Reply:
The service endpoint is also a resource, so I am not sure why exactly my predecessors added the service endpoint here. I’d be happy to drop it.

Reply:
yes, I would drop it

Reply:
Done in deb62199